foodsaving/cases/templates/conflict_resolution_case_continued.mjml

```diff
@@ -0,0 +1,30 @@
+<mjml>
+    <mj-head>
+        <mj-include path="../../base/templates/_base-head.mjml"/>
+        <mj-title>{% trans affected_user_name=affected_user.display_name|safe %}Continue the discussion about the conflict with {{ affected_user_name }}{% endtrans %}
```

Comment:
Why not 'Keep discussing'? It's shorter than 'Continue the discussion' and the same string as the one we use in the frontend.